Dockerfile.openshift.tests

```diff
@@ -4,5 +4,6 @@ COPY run-e2e-nrop-*.sh /usr/local/bin
 COPY numacell /bin
 COPY pause /
 COPY topics.json /usr/local/share
+COPY buildinfo.json /usr/local/share
```

Comment:
I believe we should have this in the operator image rather than the tests, because tests image is created always and only from the main branch and will be used to run on all releases while it would be more beneficial to get this data from the operator image itself instead

Reply:
that's a good point. It's useless in the test image.